content/blog/2018/08/2019-08-31-speaker-blog-delivery-automation.adoc

```diff
@@ -0,0 +1,82 @@
+
+Your Jenkins CI orchestration is all about workflow processing for your Dev process, but now you want to push continuous delivery.
+This of course means you need to have the pre-production and production teams on board with automation.
+Seems like an easy task, but you will soon find that cultural transformation in organizations is hard.
+Really hard.
+Whether you work in a large, small or somewhere in between organization, teams continuously struggle with changing cultural mindset to one of automation, inclusion and efficient workflow and design.
+
+
+So where do you start?
+Choosing the right tools can streamline this process, but how can you get people on board?
+And what are the risks if they choose not to follow your path to a fully automated process across the pipeline?
+
+Communicating the full benefits of what you have achieved by automating your development processes is a good place to start.
+But, by now most organizations understand the benefit of agile practices and its ability to deliver new innovation to end users at a rapid pace.
+And therein lies the real issue.
+Most organizations have not seen the promise of agile due mainly in part to downstream practices that continue to be rooted in waterfall methods.
+While agile practices have brought testing into the same room as developers, this is not always the case for production control.
+Not everyone has adopted the concept of a Site Reliability Engineer, someone who manages production, but works on the same Agile team as the developers.
+Most production deployments are still managed by throwing updates over the proverbial wall and waiting for the production team to execute their own release process to the production environment.
+And it may not be fast or agile.
+
+image::/images/post-images/2018-08-31/image1.jpg[DevOps Superhero, float="right", width=300]
+
+How can we bring production teams into the new practice of continuous delivery and deployment?
+The most critical component is understanding that production teams live by a different mantra than agile developers.
+Agile developers believe that large, monolithic quarterly releases are high risk.
+Managing 3 months’ worth of change in a single release can create a deployment headache.
+Bug discovered in production in this cycle can sometimes take months to resolve, if we even know they’re there in the first place.
+On the other side of the house, production control believes that a team that is doing daily releases must have real issues, and therefore are high risk.
+If agile practices are going to truly achieve continuous deployment, where production gets new updates as soon as they are ready, we have some work and listening to do on the production release side of the house.
+
+While we are shifting to a more DevOps approach, Production Teams have some basic requirements that Developers need to understand.

```

Comment:
First time this term is capitalized.

Also, "Developers" unnecessarily capitalized.